src/java.base/share/classes/java/util/Comparator.java

```diff
@@ -189,6 +188,44 @@ default Comparator<T> reversed() {
         return Collections.reverseOrder(this);
     }
 
+    /**
+     * Returns the greater of two values according to this comparator.
+     * If the arguments are equal with respect to this comparator,
+     * the {@code a} argument is returned.
+     *
+     * @param   a   an argument.
+     * @param   b   another argument.
+     * @param   <U> the type of the arguments and the result.
+     * @return  the larger of {@code a} and {@code b} according to this comparator.
+     * @throws  ClassCastException if the collection contains elements that are
+     *          not <em>mutually comparable</em> (for example, strings and
+     *          integers).
+     *
+     * @since 25
+     */
+    default <U extends T> U max(U a, U b) {
```

Comment:
The parameter names are o1 and o2 in the `compare` method min and max build on. 
Though a and b are used in the class javadoc example and x and y are used in the spec description.
Can we be consistent in the API?  (o1, o2) perhaps.  Someday, the parameter names *may* be more significant.

Reply:
Done, thanks.